src/WebJobs.Extensions.DurableTask/EntityScheduler/TypedInvocationExtensions.cs

```diff
@@ -30,8 +30,7 @@ public static class TypedInvocationExtensions
         /// (which is the operation name) and the argument list (which is the operation content, deserialized into
         /// an object array).
         /// </remarks>
-        public static async Task DispatchAsync<T>(this IDurableEntityContext context)
-            where T : new()
+        public static async Task DispatchAsync<T>(this IDurableEntityContext context, params object[] constructorParameters)
```

Comment:
It may be helpful to document how these parameters work. In particular in combination with dependency-injected constructor parameters. Is there a ordering requirement?